administrator/components/com_contact/src/Extension/ContactComponent.php

```diff
@@ -152,4 +156,22 @@ protected function getStateColumnForSection(string $section = null)
     {
         return 'published';
     }
+
+    /**
+     * Returns valid contexts for schemaorg
+     *
+     * @return  array
+     *
+     * @since   4.0.0
```

Comment:
```suggestion
     * @since  __DEPLOY_VERSION__
```

This way it will automatically get the correct version when it is accepted, merged and released. Sorry your mentors didnt explain this